### ISSUE TITLE
A11y audit & fixes

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks-network-catalog#31]
Prior to initial release, we should conduct a self-audit for accessibility and fix as many issues as possible. Open issues for any that cannot solved within time box.

Can use axe DevTools: https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US and ask for assistance from Ned.
